webclient/components/ptinfo-single/1time-1row-mField/vital-signs/sub-cts/weight/db/full-sync-with-db-server-mixin.js

```diff
@@ -4,7 +4,7 @@ import clientSideTable from '~/components/ptinfo-single/1time-1row-mField/vital-
 export default {
   methods: {
     async mxGetDataFromDb() {
-      const pro = await clientSideTable.api().get(clientSideTable.apiUrl + '?ptUuid=abcd')
+      const pro = await clientSideTable.api().get(clientSideTable.apiUrl)
```

Comment:
watch how reminders have been initialized. Also there is a youtube video that shows how the initialization should be done.